### DISCUSSION TITLE
Questions

### DISCUSSION THREAD

**Discussion by Cclement00** [community/community#109427] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Can you tell if I host the bot on my server it will still be hosted on the free plan,

**Comment by Void-Monarch** [comment]
what service are using , It should work as long as it within your free plan limit

**Comment by Cclement00** [comment]
It's a product service, and I want to know about the free plan I can host
as much as I want?

And I can host anytime, can you if that's possible?

Le mer. 21 févr. 2024, 08:12, VoidMonarch ***@***.***> a
écrit :
